ru/mail/chrome/communicator/wallet/SignonViewer.dtd

```diff
@@ -1,14 +1,14 @@
-<!ENTITY      windowtitle.label               "Менеджер паролей">
+﻿<!ENTITY      windowtitle.label               "Менеджер паролей">
 <!ENTITY      windowalttitle.label            "Менеджер форм: Сайты">
```

Comment:
Legacy translation, I am fixing only major and consistency issues.
This line seems OK for me, because it is not normal sentence, but UI string which consists from two parts, you can see such examples in Windows and Mac OS as well... I suggest to leave such minor issues in the legacy translation as they are.